### DISCUSSION TITLE
MMS Test V Cycle

### DISCUSSION THREAD

**Discussion by ACRW** [wenceorg/comp4187#100] [Q&A]
I've been playing around with the parameters of the solve function to try and pass the MMS convergence test. However, as finer grids are used the error comes to a plateau. When I run the test the errors and the norm errors I get are:
[1.73090129e-05, 3.00597043e-06, 4.40444635e-07, 1.03921454e-07, 3.82153453e-07]
[ 2.52562073  2.77079823  2.08346716 -1.87865854]

I don't know if the errors are in the parameters however I have tried changing them in as many ways as I can. I can only think that the error is in my V Cycle implementation. I have already used the transpose of the prolongation instead of the injection, as you mentioned in a previous post.

**Comment by annereinarz** [comment]
I assume you've already tried playing around with the the smoothing steps? Are you solving using LU on the coarsest grid? Or are you solving iteratively there?

**Reply by wence-** [reply]
It could also be that as you're going to finer grids you're no longer observing discretisation error (which should converge) but rather algebraic error (because the solver has only converged to a relative tolerance). If you tighten the tolerance of the linear solver `rtol=1e-10` (say) do you get better behaviour?
